server/internal/repository/eatery.go

```diff
@@ -55,6 +62,15 @@ func (r *Repository) GetEatery(ctx context.Context, eateryID uuid.UUID) (*Eatery
 	return eatery, nil
 }
 
+func (r *Repository) GetEateryEateryIDReviews(ctx context.Context, eateryID uuid.UUID) ([]*ReviewDetail, error) {
+	reviews := []*ReviewDetail{}
+	if err := r.db.SelectContext(ctx, reviews, "SELECT * FROM reviews WHERE id = ?", eateryID); err != nil {
```

Comment:
普通に`id`だとReviewのIDになってしまいます
```suggestion
	if err := r.db.SelectContext(ctx, reviews, "SELECT * FROM reviews WHERE eatery_id = ?", eateryID); err != nil {
```